src/com/raghav/javabasics/day7/CompareReturnNonCommonElementsArrayList.java

```diff
@@ -6,32 +6,25 @@
 public class CompareReturnNonCommonElementsArrayList {
     public static void main(String args[]) {
         CompareReturnNonCommonElementsArrayList result = new CompareReturnNonCommonElementsArrayList();
-        result.compareNonCommonElementsArrayList(Arrays.asList(new Integer[]{10, 2, 3, 4, 6}),
+        result.compareNonCommonElementsArrayList(Arrays.asList(new Integer[]{10, 2}),
                 Arrays.asList(new Integer[]{1, 2, 5, 4}));
         result.compareNonCommonElementsArrayList(Arrays.asList(new Integer[]{}), Arrays.asList(new Integer[]{1, 2, 3}));
     }
 
     private void compareNonCommonElementsArrayList(List<Integer> arrList1, List<Integer> arrList2) {
         ArrayList<Integer> arraylist = new ArrayList<>();
         for (Integer integer : arrList1) {
-            if (arrList2.contains(integer)) {
+            if (!arrList2.contains(integer)) {
                 arraylist.add(integer);
             }
         }
         for (Integer integer : arrList2) {
-            if (arrList1.contains(integer)) {
+            if (!arrList1.contains(integer)) {
                 arraylist.add(integer);
             }
         }
```

Comment:
Not necessary for the same reason as you removed (a code block) the intersection of two arrays.

Reply:
Two loops are necessary to get the non-repeating elements from the 2 arrayLists.